content/fr/_index.md

```diff
@@ -54,9 +54,9 @@ title: "InnerSource Commons"
         {{< image-community >}}
       </div>
       <div class="col-md-6">
-          <p class="section-title h2">Join the Community</p>
-          <p>We support practitioners and those who want to learn about InnerSource through a broad array of activities. To get started, join the growing ISC community via Slack and introduce yourself. You will find some great companions to join you on your InnerSource journey.</p>
-          <a href="/community/" class="btn-link">Join Community<i class="ti-arrow-right"></i></a>
+          <p class="section-title h2">Rejoindre la Communauté</p>
+          <p>Nous soutenons ceux qui partiquent et qui veulent en savoir plus sur InnerSource au travers d'un large éventail d'activités. Pour commencer, rejoignez la communauté grandissante d'ISC via Slack et présentez-vous. Vous y trouverez d'excellents contacts pour vous accompagner sur l'InnerSource.</p>
```

Comment:
partiquent => pratiquent

Reply:
good catch, taken into acount in next commit